public/v4/apps/tubesync.yml

```diff
@@ -0,0 +1,41 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: ghcr.io/meeb/tubesync:$$cap_tubesync_version
+        volumes:
+            - $$cap_appname-config:/config
+            - $$cap_appname-downloads:/downloads
+        restart: always
+        environment:
+            PUID: 1000
+            PGID: 1000
+            TZ: America/Chicago
+            HTTP_USER: $$cap_username
+            HTTP_PASS: $$cap_password
+        caproverExtra:
+            containerHttpPort: '4848'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_tubesync_version
+          label: TubeSync Version
+          defaultValue: 'v0.12.0'
+          description: Check out their docker page for the valid tags https://github.com/meeb/tubesync/tags
+          validRegex: /^([^\s^\/])+$/
+        - id: $$cap_username
+          label: TubeSync User
+          defaultValue: ''
+          description: 'Username for TubeSync'
+        - id: $$cap_password
+          label: Password
+          defaultValue: ''
+          description: 'Your password for TubeSync'
+          validRegex: /^([^\s^\/])+$/
+    instructions:
+        start: >-
+            TubeSync is a PVR (personal video recorder) for YouTube. Or, like Sonarr but for YouTube (with a built-in download client). It is designed to synchronize channels and playlists from YouTube to local directories and update your media server once media is downloaded.
+        end: >-
+            Aaaand you're done! 😄
+            Your service is available at http://$$cap_appname.$$cap_root_domain
+    displayName: 'TubeSync'
+    isOfficial: false
```

Comment:
Isn't this the official image of TubeSync? Looks like it's the same repo on github.

Reply:
Oh, I think I misunderstood the isOfficial tag. I thought it meant the one-click-app was created by the creator of the project, and since I'm not a contributor to TubeSync I thought that was false. Good to know, I'll update that and the other changes later today.

Reply:
Done